### ISSUE TITLE
Add Scars

### ISSUE THREAD

**Issue by yochaigal** [yochaigal/kettlewright#85]
It would be cool if we added another box in the character sheet for [Scars](https://cairnrpg.com/wip/2e/core-rules/#scars). It can be a drop down (not rollable) table of 12 items (see table) that the PC could choose from, appropriate to the Scar they received. It would then prefill those scars in a box. 

## A bit about Scars

When a PC loses takes exactly enough damage to drop their HP to 0, they take a Scar from the Scars table.
This is not a table you roll on, however, You actually look at the damage done to you and check the result of the row associated with that damage amount.

For example, if a character has 3 HP and takes 3 damage (and isn't wearing armor) it would bring them to exactly 0 HP.
If another character has 2 HP and is wearing 1 Armor, and takes 3 damage, it would bring them to exactly 0 HP as well.

Both characters take a Scar. However, the PC in the first example would look at the "3" row in the Scars table, while the other PC would look at the "2" row (because they lost 2 HP). It doesn't matter that the other person did 3 damage total, only the HP lost matters.

TL;DR total HP to 0 = row on the Scars table you look at.
